src/ui/components/results/alternativeverticalscomponent.js

```diff
@@ -39,7 +39,7 @@ export default class AlternativeVerticalsComponent extends Component {
      * This gets updated based on the server results
      * @type {AlternativeVertical[]}
      */
-    this.verticalSuggestions = AlternativeVerticalsComponent._buildVerticalSuggestions(
+    this.verticalSuggestions = this._buildVerticalSuggestions(
```

Comment:
This was necessary to get the componentManager. This was done in another PR by Tom.